app/grandchallenge/reader_studies/models.py

```diff
@@ -798,11 +800,16 @@ class DisplaySet(CIVForObjectMixin, UUIDModel):
     title = models.CharField(max_length=255, default="", blank=True)
 
     def __str__(self):
-        parts = [self._meta.verbose_name.title()]
-        if self.title:
-            parts.append(f"{self.title!r}")
-        parts.append(f"(ID: {self.id})")
-        return " ".join(parts)
+        result = f"{self.title!r}" if self.title else str(self.pk)
+
+        values = []
+        for value in self.values.all():
+            if value.image:
+                values.append(value.image.name)
+            if value.file:
+                values.append(os.path.basename(value.file.file.name))
+        values = ", ".join(values)
+        return f"{result} ({values})" if values else f"Empty {result}"
```

Comment:
I guess you changed this so that it's the same as for ArchiveItems? Would be worth refactoring as well then. 

I'm not 100% sure about the format though. Do we really want to add all the values in the title? There is no limitation on how many values an item can have, this could get very long. Also, the pk is only added if there is no title. I feel like that pk should always be there, right?

Reply:
I've opted to standardize things a bit. Each CIV is now shown as an interface.title + title, this was already the case for the list view but I copied this to the other representations.

<img width="1256" alt="civ_set_list_deletion" src="https://github.com/comic/grand-challenge.org/assets/7871310/a5fc4537-fc1d-4489-801f-21fd413610f0">


Furthermore, I've taken the liberty of updating the single civset deletion template it can have all the CIVs shown in a data table format:

<img width="1357" alt="single_civ_deletion" src="https://github.com/comic/grand-challenge.org/assets/7871310/3815565a-31a9-4a51-abd5-17bd22841656">

Reply:
Note: I've copied the Warning to the list deletion as well.